content/en/docs/about/membership.md

```diff
@@ -0,0 +1,214 @@
++++
+title = "Community Membership"
+description = "Guidelines for contributing to Kubeflow"
+weight = 30
+aliases = ["/docs/membership/"]
++++
+
+
+{{< note >}}
+This document is a work in progress
```

Comment:
do you want to remove this?

Reply:
The intent was to say 'this document is not final", but I'm good with removing it if this muddies the message